src/librustdoc/clean/utils.rs

```diff
@@ -624,3 +624,24 @@ where
     *cx.impl_trait_bounds.borrow_mut() = old_bounds;
     r
 }
+
+crate fn find_closest_parent_module(tcx: TyCtxt<'_>, def_id: DefId) -> Option<DefId> {
+    if item.is_fake() {
+        // FIXME: is this correct?
+        None
```

Comment:
I think this is probably correct; at very least, there's no other reasonable behavior.
```suggestion
        None
```